noxfiles/utils_nox.py

```diff
@@ -60,3 +60,39 @@ def teardown(session: nox.Session) -> None:
 def install_requirements(session: nox.Session) -> None:
     session.install("-r", "requirements.txt")
     session.install("-r", "dev-requirements.txt")
+
+
+@nox.session()
+def init_saas_connector(session: nox.Session) -> None:
+    connector_name = session.posargs[0].replace(" ", "")
+    connector_id = "_".join(session.posargs[0].lower().split(" "))
+    variable_map = {"connector_name": connector_name, "connector_id": connector_id}
+
+    # create empty config and dataset files
+    open(f"data/saas/config/{variable_map['connector_id']}_config.yml", "w")
```

Comment:
i get a little worried here, and below, about the edge case (doesn't feel like too much of an edge case to me) where this is accidentally run against a set of files that already exists but hasn't been committed to version control yet. we'd end up overwriting the config, dataset, fixtures and tasks files, which could have had updates made to them. i'm thinking something like - i run `nox -s init_saas_connector -- Wahoo`, i do some work to fill out my starter files, then i accidentally run `nox -s init_saas_connector -- Wahoo` again, for whatever reason.

what do you think about being a bit more defensive here - identifying if these files already exist, and erroring out with a clear error message if so (or at least not overwriting whichever files do exist)?

Reply:
I'll go ahead and add a check here